README.adoc

```diff
@@ -57,12 +57,18 @@ Try to use at least version `0.11.0` as we haven't tested another version.
 Assuming you have Go installed and configured (have `$GOPATH` setup) here is
 how to build.
 
-Check out the code
+Check out the code. On Linux or Mac:
 
 ----
 $ git clone https://github.com/almighty/almighty-core $GOPATH/src/github.com/almighty/almighty-core
 ----
 
+On Windows PowerShell:
+
+----
+$ git clone https://github.com/almighty/almighty-core $env:GOPATH\src\github.com\almighty\almighty-core
+----
```

Comment:
In the absence of any recommended path for getting started on Windows, I just made my own way, and like any good open source citizen wanted to contribute my path back.

That tells me that the real issue here is the lack of described path for windows...